nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/LogAttribute.java

```diff
@@ -123,12 +133,26 @@ protected void init(final ProcessorInitializationContext context) {
     protected String processFlowFile(final ProcessorLog logger, final DebugLevels logLevel, final FlowFile flowFile, final ProcessSession session, final ProcessContext context) {
         final Set<String> attributeKeys = getAttributesToLog(flowFile.getAttributes().keySet(), context);
         final ProcessorLog LOG = getLogger();
+        final String dashedLine;
+
+        String logPrefix = context.getProperty(LOG_PREFIX).evaluateAttributeExpressions().getValue();
```

Comment:
My apologies, I led you slightly astray in method mentioned before.  The evaluateAttributeExpressions seems like it would benefit from making use of the current flowfile,  #evaluateAttributeExpressions(FlowFile flowfile).  Otherwise, everything looks great.  If you would like me to make this change on merge, let me know.  Otherwise, that would be the last adjustment needed to call this complete.